dispatcher/src/conf.dispatcher.d/available_farms/wknd.farm

```diff
@@ -90,9 +90,16 @@
 		# Marketing parameters can normally be ignored on most websites as they are tracked
 		# through different means. 
 		/ignoreUrlParams {
+                   # Common rule denying all parameters
 			/0001 { /glob "*" /type "deny" }
-			# $include "../cache/marketing_query_parameters.any"
-		}
+
+                   # Test-specific rules
```

Comment:
I think we can drop the test-specific rules here. `utm_campaign` is duplicate as it's already defined in the marketing parameters file and the `page` one is not used in wknd tutorial and would already be covered by the `*` rule in `/0001`.